react/features/app/actions.js

```diff
@@ -6,11 +6,20 @@ import { APP_WILL_MOUNT, APP_WILL_UNMOUNT } from './actionTypes';
 import {
     _getRoomAndDomainFromUrlString,
     _getRouteToRender,
+    areRoutesEqual,
     init
 } from './functions';
 import './reducer';
 
 /**
+ * Variable saving current route of the app. Later it will be extracted as
+ * a property of the class with Router specific logic.
+ *
+ * @type {Object}
+ */
+let currentRoute = {};
```

Comment:
1. No global state please! Either React Component state or Redux store.

2. AbstractApp's `state` already tracks the current route. It has a `route` property.